packages/lib/app/environment.ts

```diff
@@ -13,7 +13,7 @@ interface KeycloakRef {
 }
 
 export default function buildEnvironment(
-  keycloakRef: KeycloakRef,
+  keycloakRef?: KeycloakRef,
```

Comment:
Making keycloak ref optional for environments in getStaticProps. Open to suggestions! I could be doing this incorrectly.